streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -126,15 +126,15 @@ public StateStore getStateStore(final String name) {
     }
 
     @Override
-    public <K, V> void forward(final K key,
-                               final V value) {
+    public <K1 extends K, V1 extends V> void forward(final K1 key,
+                               final V1 value) {
```

Comment:
nit: fix indention (similar below)

Reply:
Ah, forgot to reformat. Thanks for the catch.